src/Tool-DependencyAnalyser/DAUnkownManuallyResolvedPackage.class.st

```diff
@@ -0,0 +1,37 @@
+"
+I am a Warning raised to the user if the list of manually resolved dependencies contains a package that does not exist in the system.
+"
+Class {
+	#name : #DAUnkownManuallyResolvedPackage,
```

Comment:
typo `DAUnkownManuallyResolvedPackage` shoud be `DAUnknownManuallyResolvedPackage`